tool/tctl/common/kube_command.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"os"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service"
+)
+
+// KubeCommand implements "tctl kube" group of commands.
+type KubeCommand struct {
+	config *service.Config
+
+	// format is the output format (text or yaml)
+	format string
+
+	// verbose sets whether full table output should be shown for labels
+	verbose bool
+
+	// kubeList implements the "tctl kube ls" subcommand.
+	kubeList *kingpin.CmdClause
+}
+
+// Initialize allows KubeCommand to plug itself into the CLI parser
+func (c *KubeCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	c.config = config
+
+	kube := app.Command("kube", "Operate on registered kubernetes clusters.")
+	c.kubeList = kube.Command("ls", "List all kubernetes clusters registered with the cluster.")
+	c.kubeList.Flag("format", "Output format, 'text', or 'yaml'").Default("text").StringVar(&c.format)
```

Comment:
Can we add JSON to the mixture of formats? Some other commands support it, and I believe it is easier to process programmatically by CLI tools.